bitbots_localization/cfg/localization.cfg

```diff
@@ -74,7 +67,7 @@ group_pf.add("lines_factor", double_t, 0, "how much line measurements are contri
 group_pf.add("goals_factor", double_t, 0, "how much goal measurements are contribute to the particle weight", None, min=0.0, max=1)
 group_pf.add("field_boundary_factor", double_t, 0, "how much field boundary measurements contribute to the particle weight", None, min=0.0, max=1)
 group_pf.add("corners_factor", double_t, 0, "how much corners measurements contribute to the particle weight", None, min=0.0, max=1)
-group_pf.add("tcrossings_factor", double_t, 0, "how much tcrossingd measurements contribute to the particle weight", None, min=0.0, max=1)
+group_pf.add("t_crossings_factor", double_t, 0, "how much tcrossingd measurements contribute to the particle weight", None, min=0.0, max=1)
```

Comment:
tcrossingd -> t-crossing

Reply:
I think we only use underscores in the parameter names

Reply:
I meant the comment.

Reply:
Ah okay sorry